terra-jupyter-aou/Dockerfile

```diff
@@ -42,7 +42,7 @@ RUN mkdir -p /etc/spark/conf.dist && mkdir -p /etc/hadoop/conf.empty && mkdir -p
     && update-alternatives --install /etc/hadoop/conf hadoop-conf /etc/hadoop/conf.empty 100 \
     && update-alternatives --install /etc/hive/conf hive-conf /etc/hive/conf.dist 100
 
-ENV HAIL_VERSION=0.2.57
+ENV HAIL_VERSION=0.2.61
```

Comment:
I squeezed in a hail upgrade too to match `terra-jupyter-hail` - let me know if you're not ok with this. :) I built and ran the image locally and did some minimal testing

Reply:
Sounds good - thank you.